unit_tests/routers/test_event.py

```diff
@@ -1,4 +1,5 @@
 from fastapi import status
+from fastapi.encoders import jsonable_encoder
```

Comment:
Ooo nice - didn't know about these

Reply:
It was a necessity - if you pass a Pydantic model to FastAPI via JSON with data types that aren't inherently serialisable to JSON such as Python datetime, the tests will fail. Fun nugget of info for this afternoon!